earthpy/tests/test_plot.py

```diff
@@ -88,3 +87,29 @@ def test_single_band_2dims():
     assert arr.ndim == 2
     assert len(fig.axes[0].get_images()) == 1
 
+""" Colorbar Tests """
+
+
+im_arr2 = np.random.randint(10, size=(5, 5))
```

Comment:
It seems like `im_arr2`, `fig`, `ax`, and `im` could be created within the `test_colorbar_height` function instead, since they are only used there. 

Also (minor nits to pick):

- Is there a better name than `im_arr2`?
- Is specifying a `cmap` necessary for the test?

Reply:
i've created a fixture for the arr. I think it's a good idea to specify a cmap because it just is there for me to know colorbar works. all of the testing of creating the cbar should happen in mpl. But i think testing that the arguments that you think work still work is good even if there isn't an explicit test. i'd like to keep the cmap argument. i suppose i could create a new test for it but that is not a good idea because it opens a can of works of testing colorbar which again is a mpl function that is already (i hope) tested.

Reply:
Cool - thanks for clarifying!